Tools/MobileMiniBrowser/MobileMiniBrowserFramework/WebViewController.m

```diff
@@ -139,8 +139,9 @@ - (IBAction)showTabs:(id)sender
 
 - (IBAction)showSettings:(id)sender
 {
+    UIPopoverPresentationController *presentationController = [self.settingsViewController popoverPresentationController];
+    presentationController.barButtonItem = sender;
```

Comment:
It's not a good policy to assume that `sender` is a button. If this were ever hooked up to some other UI (e.g. a menu item) that assumption would be invalid.

Better to add an IBOutlet for the button.